examples/integration_with_error.py

```diff
@@ -10,6 +10,8 @@
 from pyspecProcScripts import integrate_limits, integral_w_errors
 from numpy.random import seed
 
+# sphinx_gallery_thumbnail_number = 2
+
```

Comment:
I would recommend generally trying to choose representative figures for each example -- here, it shows up as figure 2 on the command line (this is explained in sphinx-gallery docs)